lib/shared/permissions.js

```diff
@@ -101,7 +116,22 @@ exports.getEnvironmentCommandPrefix = (environment) => {
     }
 
     if (isWindows) {
-      return [ 'set', `${key}=${value}`, '&&' ];
+      const definition = `${key}=${value}`;
+      return [
+        'set',
+        _.attempt(() => {
+          if (/\s/.test(definition)) {
+            return quoteString(definition);
+          }
+
+          return definition;
+        }),
+        '&&'
+      ];
+    }
+
+    if (/\s/.test(value)) {
```

Comment:
Sounds good, I'll make sure they are always quoted. @jhermsmeier sadly `command-join` doesn't play nice with environment variable declarations. In the case of Windows, for example, `set` shouldn't be quoted, otherwise `cmd.exe` complains.